torch/quantization/fx/quantization_patterns.py

```diff
@@ -1456,12 +1459,16 @@ def convert(self,
 @register_quant_pattern(torch.nn.functional.hardtanh)
 @register_quant_pattern(torch.nn.functional.hardtanh_)
 @register_quant_pattern(torch.nn.functional.interpolate)
+@register_quant_pattern(torch.nn.functional.max_pool1d)
```

Comment:
For my understanding, the CopyNodeQuantizeHandler is used for many patterns. How is it that only maxpool1d/2d/3d produces a pattern with a dequant->maxpool->quant? What about other operators like clamp?

Reply:
Yeah I think we should probably remove GeneralTensorValueQuantizeHandler and move everything here, it looks like some backend may not work when we have "quant - some_general_op - dequant", TensorRT included. I think we should just produce "dequant - float_op - quant" pattern for all these ops.
Planning to change that in a separate PR, current goal is to test TensorRT integration.